manifest/x86_64/z/zed.filelist

```diff
@@ -1,5 +1 @@
 /usr/local/bin/zed
-/usr/local/libexec/zed-editor
-/usr/local/share/applications/zed.desktop
-/usr/local/share/icons/hicolor/1024x1024/apps/zed.png
-/usr/local/share/icons/hicolor/512x512/apps/zed.png
```

Comment:
Why was the desktop icon removed?

Reply:
Because we don't use it?

It's a minimal full compile instead of using the pre-built binaries.

Reply:
Rebuilding with the png files...